### PR TITLE
Print errors in create_final in red (not green)

### PR DESCRIPTION
**Pull request by jwalton3141**
Addresses #44 

Before:

After (not perfect, but better)

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
-# jrNotes 0.7.3 _2020-09-15_
+# jrNotes 0.7.4 _2020-09-18_
+  * Bug: Errors hit in `create_final` should be printed in red (not green)
```
Comment (csgillespie) [inline NEWS.md:2]: Our general style is to try and use `()` for functions, i.e. `create_final()`. Picky I know, but it reinforces what we do in the notes.

**Review by csgillespie — CHANGES_REQUESTED** [review]
Very picky comment. After that, I'm happy

**Comment by jwalton3141** [comment]
Is this better or worse (image below)

In general we have colour leakage in _every_ call to `msg_xxx`. Though we don't normally notice it because each message starts with a new colour. 

Really `crayon::reset` should be added to the end of every message. Doing so would style `msg_error` with `stop = TRUE` as:

image

File: NEWS.md (hunk 1 above)
Comment (jwalton3141) [reply NEWS.md:2]: Agreed, I also prefer using `()` to indicate functions. Consider this a slip up on my usual high standards ;)

**Comment by JohnMcIntyreJR** [comment]
New one looks more professional but original looks more eye catching and will probably alert user more @jwalton3141

**Comment by jwalton3141** [comment]
The colouring before was actually the result of a bug (unless the colours were intentionally not closed ;)).

**Comment by jwalton3141** [comment]
#7488439 intentionally does not close the colours **after a call to stop**, which gives a stop message as
image

**Review by JohnMcIntyreJR — APPROVED** [review]

**Pull request merged**